ui/components/app/confirm/info/row/copy-icon.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useCallback, useState } from 'react';
+
+import { IconColor } from '../../../../../helpers/constants/design-system';
+import { Icon, IconName, IconSize } from '../../../../component-library';
+
+export const CopyIcon: React.FC<{ copyText: string }> = ({ copyText }) => {
+  const [success, setSuccess] = useState(false);
+
+  const copyToClipboard = useCallback(async () => {
```

Comment:
Should we use the existing `useCopyToClipboard` hook for this so there is a single definition in the UI?

Reply:
Good point, I updated the PR